source/fem-dem/vans_assemblers.cc

```diff
@@ -89,10 +89,13 @@ VANSAssemblerCoreModelB<dim>::assemble_matrix(
       const double tau =
         this->simulation_control->get_assembly_method() ==
             Parameters::SimulationControl::TimeSteppingMethod::steady ?
-          1. / std::sqrt(std::pow(2. * u_mag / h, 2) +
-                         9 * std::pow(4 * kinematic_viscosity / (h * h), 2)) :
-          1. / std::sqrt(std::pow(sdt, 2) + std::pow(2. * u_mag / h, 2) +
-                         9 * std::pow(4 * kinematic_viscosity / (h * h), 2));
+          1. / std::sqrt(Utilities::fixed_power<2>(2. * u_mag / h) +
+                         9 * Utilities::fixed_power<2>(4 * kinematic_viscosity /
+                                                       (h * h))) :
+          1. / std::sqrt(Utilities::fixed_power<2>(sdt) +
+                         Utilities::fixed_power<2>(2. * u_mag / h) +
+                         9 * Utilities::fixed_power<2>(4 * kinematic_viscosity /
+                                                       (h * h)));
```

Comment:
We have a function now to calculate the stabilization. [You can see it here](https://github.com/chaos-polymtl/lethe/blob/master/include/solvers/stabilization.h#L30)